jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -141,12 +168,12 @@ void handleDeprecatedParameters(BuildLogger logger) {
   @Parameter(defaultValue = "${session}", readonly = true)
   MavenSession session;
 
-  @Parameter private FromConfiguration from = new FromConfiguration();
+  @Parameter private final FromConfiguration from = new FromConfiguration();
```

Comment:
nit: final variables up top - also, it looks like the `handleDeprecatedParameters` method is out of place, and the package-private `session` should be above private `project`.